libs/acn/RootPDU.h

```diff
@@ -33,8 +33,8 @@ namespace acn {
 
 class RootPDU: public PDU {
  public:
-  explicit RootPDU(unsigned int vector):
-    PDU(vector),
+  explicit RootPDU(unsigned int vector, bool force_length_flag = false):
+    PDU(vector, FOUR_BYTES, force_length_flag),
```

Comment:
Just a question: What are the "FOUR_BYTES"? Some kind of magic/FourCC?

Reply:
Not really, it's just an internal enum for the width of the vector, defined here:
https://github.com/OpenLightingProject/ola/blob/e7b403927ad462d6ba4eed7d72cdd3e1fb1fa4e8/libs/acn/PDU.h#L39-L43

Just a bit more user-friendly than magic numbers like 4 appearing all over the code with no context.

Reply:
Okay, thanks for explaining :)